app/src/main/res/layout/activity_child_login.xml

```diff
@@ -90,12 +90,20 @@
                 <TextView
                     android:id="@+id/lblVersion"
                     android:layout_width="match_parent"
-                    android:layout_height="40dp"
-                    android:layout_weight="1"
+                    android:layout_height="wrap_content"
                     android:text="@string/app_version"
                     android:textAlignment="center"
                     android:textColor="@android:color/white"
                     android:textSize="10sp" />
+
+                <TextView
+                    android:id="@+id/tv_lastDateSynced"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:hint="Last Sync Date: 10/10/19"
+                    android:textAlignment="center"
+                    android:textColor="@android:color/white"
+                    android:textSize="10sp" />
             </LinearLayout>
```

Comment:
@ChiragAwale add this text view in other activity_login files too..
<img width="480" alt="Screen Shot 2019-10-14 at 07 18 05" src="https://user-images.githubusercontent.com/9145217/66725547-ed735b80-ee52-11e9-9324-992aaebc0e05.png">

Reply:
@rrijal53 gotcha